lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -60,7 +60,7 @@
                 (select)="onEnterKeyPressed(row, $event)"
                 (keyup)="onRowKeyUp(row, $event)"
                 (keydown)="onRowEnterKeyDown(row, $event)"
-                [adf-upload]="allowDropFiles && rowAllowsDrop(row)"
+                [adf-upload]="((isFolder(row) && allowDropFiles) || (!isFolder(row) && allowDropFileVersion)) && rowAllowsDrop(row)"
```

Comment:
looks quite complicate assertion for a template maybe is better incapsulate in  a method